scanner/src/lib/dataproducts/Battery_ManufacturingDataSheet_v0.1.svelte

```diff
@@ -210,56 +217,28 @@
     link
     value={data.legalConformity?.conformityDeclaration}
   />
-</article>
+</Article>
 
 <style lang="scss">
-  article {
-    color: white;
-    font-style: normal;
-
-    .title {
-      font-size: 1rem;
-      font-weight: 500;
+  .act-compliance {
+    display: flex;
+    align-items: center;
+    margin-bottom: 1rem;
+    p {
+      flex-grow: 0;
+      flex-shrink: 0;
+      flex-basis: 45%;
       line-height: 1.5rem;
-      margin-bottom: 1rem;
-
-      &.no-bottom-margin {
-        margin-bottom: 0;
-      }
+      padding-right: 0.5rem;
     }
 
-    .subtitle {
-      font-size: 0.75rem;
-      font-weight: 400;
-      color: white;
-      margin-bottom: 1rem;
-    }
-
-    .divider {
-      border-bottom: 1px solid #20303e;
-      margin-bottom: 1rem;
-    }
-
-    .act-compliance {
-      display: flex;
-      align-items: center;
-      margin-bottom: 1rem;
-      p {
-        flex-grow: 0;
-        flex-shrink: 0;
-        flex-basis: 45%;
-        line-height: 1.5rem;
-        padding-right: 0.5rem;
-      }
-
-      .compliance-img {
-        width: 1.5rem;
-        line-height: 1.5rem;
-      }
+    .compliance-img {
+      width: 1.5rem;
+      line-height: 1.5rem;
```

Comment:
This looks like an SVG, which for some reason is an `<img src>` instead of a component, and then has just `width` but no `height` - I'm not sure this works on Safari, please test.

Reply:
Actually hm, no, this selector is for the div around the img, anyway please test on Safari to confirm